docs/mydoc.md

```diff
@@ -0,0 +1,84 @@
+<style>
+</style>
+
+# تجربتي مع منصة Crowdin للترجمة
+
+تمّ إنشاء **مجتمع العلوم المفتوحة** في السعودية [OSCSA ](https://osc-ksa.com/)تماشيًا مع رؤية السعودية 2030 لدعم وتوفير بنية تحتية للعلوم المفتوحة في العالم العربي من خلال العمل مع مجتمعات العلوم المفتوحة الأخرى مثل  Turing Way  باستخدام  [GitHub](www.github.com) ومنصة [Crowdin](www.crowdin.com)
+
+موقع Crowdin عبارة عن *منصة تعريب* تقدم خدمة ترجمة المواقع إلى عدة لغات بشكل احترافي وجماعي دون ضياع الجهد أو تداخله عن طريق تقاسم المهام بين المترجمين.
+
+- بدايةً قُم بإنشاء حساب في المنصة عن طريق التسجيل من خلال البريد الالكتروني (أو عبر وسائل التواصل الاجتماعي).
+  
+![1](https://user-images.githubusercontent.com/31999574/163883660-c793832f-43c9-45e4-836d-2d53e90ff318.png)
+  
+  ---
+  
+
+- سيتمً إرسال رسالة تأكيد إلى بريدك الالكتروني لتفعيل حسابك على الموقع.
```

Comment:
alignment should be from right to left - can you fix it, if possible, please?

Reply:
whenever I preview the code the alignment is correct but once I save it it goes to the left again(lines starts with *) so I had to remove bullet list after I had trying to solve it without any success.